openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1117,6 +1118,53 @@ Cache-Control: no-store
 }
 ```
 
+# Callback Endpoint {#callback_endpoint}
+
+This endpoint is used to receive notification from the Wallet whether credential has been successfully received in the Wallet or not. Support for this endpoint is OPTIONAL.
+
+This endpoint can be used after the Credential Issuer has sent Credential Response or Batch Credential Response. It enables the Credential Issuer to take subsequent actions after issuance, depending on whether the credential has been accepted and successully stored by the Wallet, rejected by the Wallet, or errors and other unforeseen circumstances have occurred during the Wallet's processing.
+
+Communication with the Callback Endpoint MUST utilize TLS.
+
+## Callback from the Wallet
+
+The Wallet sends a callback to the Callback Endpoint by sending the following parameters in the entity-body of an HTTP POST request using the `application/json` media type.
+
+* `status`: REQUIRED. Status whether the credential issuance was successful or not. The value MUST be either `success` or `error`.
```

Comment:
"error": true,
"error_description": "that long story"

Implementations may benefit by using boolean values and key names pairwised with their scope

Reply:
"error": true,
"error_description": "that long story"

Implementations may benefit by using boolean values and key names pairwised with their scope

Reply:
There was an exchange on this on Bitbucket: https://bitbucket.org/openid/connect/pull-requests/608#comment-419576731

Why do you think boolean is better than two status codes..?

Reply:
because `if true` is more explicit

let's try to follow this path:
 - if there's an error ... then ...

that's why I suggest error:true and error_description, the first filters the second and we just want to now if the flow failed or not, since we don't assume that the status might have more than two values, so a selection of multiple statuses may exceeds the requirements

Reply:
may as well follow the path:
- if `status` is ... then ...